qrcode/image/pil.py

```diff
@@ -11,20 +14,17 @@ class PilImage(qrcode.image.base.BaseImage):
 
     def new_image(self, **kwargs):
         if not Image:
```

Comment:
How would this be reachable in the first place? `Image` is imported unconditionally from `PIL`...